### DISCUSSION TITLE
Frigate and HA on the same IP address with Reolink

### DISCUSSION THREAD

**Discussion by CMDR-Sloma** [blakeblackshear/frigate#15402] [Ask A Question]
Is there a way to configure Frigate to use entities from Home Assistant and not direct connections from the cameras? 

The reolink cameras don't like multiple connections from the same IP address and eventually drops them.

**Comment by NickM-27** [comment]
The recommendation is to have frigate connect using go2rtc and have everything else read from the go2rtc proxy that way only one connection is made to the camera. 

I believe there is a way to have go2rtc read from HA but it is more complicated

**Reply by CMDR-Sloma** [reply]
I'd prefer to keep HA a central as possible and not rely on Frigate as a crucial component. This is why I am trying to find a solution for the second option.
